galaxy_api/api/ui/serializers/collection.py

```diff
@@ -50,42 +61,46 @@ class CollectionVersionBaseSerializer(serializers.Serializer):
     namespace = serializers.CharField()
     name = serializers.CharField()
     version = serializers.CharField()
-    created_at = serializers.DateTimeField(source='_created')
 
+    created_at = serializers.DateTimeField(source='_created')
 
 class CollectionLatestVersionSerializer(CollectionVersionBaseSerializer):
     metadata = CollectionMetadataBaseSerializer(source='*')
 
+class CollectionLatestVersionDetailSerializer(CollectionLatestVersionSerializer):
+    docs_blob = serializers.JSONField()
+    contents = serializers.ListField(ContentSerializer())
```

Comment:
I need `contents` to get returned in the list view as well as the detail view.

Reply:
Doesn't https://github.com/ansible/galaxy-api/pull/85/commits/e371142d13d8449fd4dae7b90b75855876c6ede1  fix this?

(not sure why, but a comment mentioning that was deleted)

Reply:
@newswangerd  https://gist.github.com/alikins/9d224e5748a1d56c92b6eb99616e3a6f should fix this. Example output https://gist.github.com/alikins/9d224e5748a1d56c92b6eb99616e3a6f